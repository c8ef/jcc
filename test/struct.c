struct S {
  int x;
  int y;
};
