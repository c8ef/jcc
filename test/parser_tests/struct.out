FunctionDecl: main
  Args[0]
  CompoundStatement
    DeclStatement
      VarDecl: s
